tests/components/zeroconf/test_init.py

```diff
@@ -78,6 +79,19 @@ async def test_setup(hass, mock_zeroconf):
     assert len(mock_config_flow.mock_calls) == expected_flow_calls
 
 
+async def test_setup_with_default_interface(hass, mock_zeroconf):
+    """Test default interface config."""
+    with patch.object(hass.config_entries.flow, "async_init"), patch.object(
+        zeroconf, "ServiceBrowser", side_effect=service_update_mock
+    ):
+        mock_zeroconf.get_service_info.side_effect = get_service_info_mock
+        assert await async_setup_component(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)